packages/web3-eth/test/integration/rpc.test.ts

```diff
@@ -264,6 +264,91 @@ describe('rpc', () => {
 			validateTransaction(res as TransactionInfo);
 			expect(res?.hash).toBe(receipt.transactionHash);
 		});
+		it('check get transaction fields', async () => {
```

Comment:
add tests to check fields exists and that no errors to get them for each tx type